scripts/bot-deployment/DeployReporterCR.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+set -e
+
+if [ $# -ne 1 ]; then
+    echo "Incorrect number of arguments supplied! First and only argument is bot's name. From this the bot's config."
+    echo "example: ./DeployBotCR.sh ethbtc-mainnet-reporter"
```

Comment:
Can you include the command, either in an `echo` or in comments, to display all GCR instances available (i.e. which configs can be pulled down)?

Reply:
This pulls the config from the cloud bucket as described in the `Bot Configs` playbook document. I added a comment to (via an `echo`) to explain how to list these.

Reply:
I also added this to the other deployment script for consistency

Reply:
I also added this to the other deployment script for consistency

Reply:
Thanks! You mispelled "available"

Reply:
@nicholaspai is the ultimate spell checker.

Reply:
LOL